index.bs

```diff
@@ -630,6 +630,33 @@ spec:url; type:dfn; text:scheme
     portals-cross-origin-load.sub.html
   </wpt>
 
+  <section algorithm="htmlportalelement-activation-behavior">
+    A <{portal}> element's [=activation behavior=] is to run the following steps:
+
+    1. Let |portalBrowsingContext| be the [=guest browsing context=] of [=this=].
```

Comment:
We're not in a Web IDL method, so `[=this=]` doesn't exist. Probably it's best to say "A `portal` element _el_'s activation behavior is to run the following steps:" and then use _el_ throughout.

Reply:
Done.